Frameworks/UIKit/UIView.mm

```diff
@@ -198,24 +199,68 @@ - (bool)_processGesturesForTouch:(UITouch*)touch event:(UIEvent*)event touchEven
         }
     }
 
-    // gesture priority list
-    const static id s_gesturesPriority[] = {[UIPinchGestureRecognizer class],
-                                            [UISwipeGestureRecognizer class],
-                                            [UIPanGestureRecognizer class],
-                                            [UILongPressGestureRecognizer class],
-                                            [UITapGestureRecognizer class] };
```

Comment:
So I previously wrote some simple tests to see what happens on iOS with the gesture lists.  From what I can tell the order in the list gives the priority.  The gesture delegates and other callbacks (e.g. simultaneous recognizing, prevention, etc) help control the recognition.  I haven't implemented the full delegate/coordination api across all gestures.  We do not use all of them, do you have a full test suite?  I think this implementation is a potential step in the right direction since we rely on creating our own gestures.  Any thoughts on next steps?